_topics/en/op_codeseparator.md

```diff
@@ -0,0 +1,150 @@
+---
+title: OP_CODESEPARATOR
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **OP_CODESEPARATOR** is an opcode that changes what data is used when
+  a signature commits to a script.  The opcode has been available since
+  the original version of Bitcoin Script, but its use and behavior have
+  changed over time (with further changes having been proposed).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  In Bitcoin 0.1, each spend's scriptSig and outpoint's scriptPubKey
+  were evaluated together with a [virtual][virtual codesep]
+  `OP_CODESEPARATOR` placed between them.  For example, using a P2PK
+  script:
+
+      <signature> OP_CODESEPARATOR <public key> OP_CHECKSIG
+
+  In legacy Bitcoin Script, such as the above example,
+  `OP_CODESEPARATOR` removes itself and any preceding parts of a script
+  from the *scriptCode*---which is part of the data that is hashed to
```

Comment:
s/which is part of the data/which is the part of the data/ ?

Reply:
I think the current text is correct.

Reply:
ok!